README.md

```diff
@@ -19,26 +19,56 @@ func main() {
 }
 ```
 
-*Note: v2 requires Go v1.18 for generics.* If you need to support Go 1.17 or
-below, you can use v1.
+> [!NOTE]
+>
+> - _v3 requires Go v1.23_ - If you need to support Go 1.18-1.22, you can use v2.
+> - _v2 requires Go v1.18 for generics_ - If you need to support Go 1.17 or below, you can use v1.
 
 Internally an `*OrderedMap` uses the composite type
 [map](https://go.dev/blog/maps) combined with a
 trimmed down linked list to maintain the order.
 
 ## Iterating
 
-Be careful using `Keys()` as it will create a copy of all of the keys so it's
-only suitable for a small number of items:
+The following methods all return
+[iterators](https://go.dev/doc/go1.23#iterators) that can be used to loop over
+elements in an ordered map:
+
+- `All()` - (Aliased to `AllFromFront`)
```

Comment:
Remove this line and fix the reference to the snippet below.

Reply:
Whoops, fixed